src/Handler/DateHandler.php

```diff
@@ -285,10 +298,14 @@ private function getDeserializationFormats(array $type): array
             return is_array($type['params'][2]) ? $type['params'][2] : [$type['params'][2]];
         }
 
-        return [$this->getFormat($type)];
+        if (isset($type['params'][0])) {
+            return is_array($type['params'][0]) ? $type['params'][0] : [$type['params'][0]];
```

Comment:
Looking at the old code:
```php
    private function getFormat(array $type): string
    {
        return $type['params'][0] ?? $this->defaultFormat;
    }
```
`$type['params'][0]` will be always `string`.

Reply:
Yeah, that's true => I removed redundant code, thanks